packages/table/src/Table.js

```diff
@@ -307,7 +307,7 @@ const getDataCell = (field, dataRow, searchText, i, onRowClick) => {
   const cellText = field.searchable ? (
     <Highlighter
       searchWords={searchText.split(/\s+/)}
-      textToHighlight={`${dataRow[dataKey]}`}
+      textToHighlight={`${dataRow[dataKey] || ''}`}
```

Comment:
This will also replace a value of `0` with an empty string, but since `searchable` is only applied to HGVSC/P and consequence columns, this should be fine.